Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM debian:stretch
+MAINTAINER James McFeeters "jmcfeeters@usgs.gov"
+RUN apt-get update && apt-get install -y \
+    python3-pip \
+    nginx-light \
+    curl \
+    apt-transport-https \
+    git
+
+COPY . /app
+WORKDIR /app
+
+RUN cp DOIRootCA2.cer /etc/ssl/certs/DOIRootCA2.pem \
```

Comment:
I'm not sure this will have the desired effect of copying the root DOI certificate from the host to the Docker container. This command will copy a `DOIRootCA2.cer` that's already in the container to `/etc/ssl/certs/DOIRootCA2.pem`. Using `COPY` should give the desired effect.